INSTALL

```diff
@@ -486,6 +486,13 @@ MFEM_USE_CEED = YES/NO
    library for performant high-order operator evaluation developed by the Center
    for Efficient Exascale Discretizations in the Exascale Computing Project.
 
+MFEM_USE_MKL_CPARDISO = YES/NO
+   Enables the interface to the Intel MKL Parallel Direct Sparse Solver for
+   Clusters. Make sure to set the correct values for MKL_MPI_WRAPPER and
+   MKL_LIBRARY_SUBDIR as shown in defaults.mk. If you configure MFEM with
+   MFEM_USE_LAPACK=YES, verify that the MKL LAPACK libraries are used. The
+   OpenMP capabilities are disabled at link time.
+
```

Comment:
Can you clarify the CPARDISO portion of the name, e.g. "the Intel MKL Parallel Direct Sparse Solver for Clusters (CPardiso)"?

What's the relation between the MKL CPARDISO and just PARDISO (https://www.pardiso-project.org/)?

Reply:
MKL CPardiso and the parallel version of Pardiso are developed independently.